gormsupport/db_test_suite.go

```diff
@@ -28,9 +29,11 @@ type DBTestSuite struct {
 // SetupSuite implements suite.SetupAllSuite
 func (s *DBTestSuite) SetupSuite() {
 	resource.Require(s.T(), resource.Database)
-	var err error
-	if err = configuration.Setup(s.configFile); err != nil {
-		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+	configuration, err := config.NewConfigurationData("")
```

Comment:
Actually it doesn't make any difference since the default values in the code is same as that in `../config.yaml' , but I added it back anyway since this isn't the PR to address that, really :)
Thanks, done here https://github.com/almighty/almighty-core/pull/868/commits/489c2202b7aae8b8a105ce296b0acfa0857eab08